handlers/handlers.go

```diff
@@ -1,14 +1,25 @@
 package handlers
 
 import (
+	"bytes"
 	"net/http"
-	"strings"
 
-	"github.com/vocdoni/blind-csp/csp"
-	"github.com/vocdoni/blind-csp/handlers/idcathandler"
-	"github.com/vocdoni/blind-csp/handlers/rsahandler"
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/types"
 )
 
+// AuthFunc is the function type required for performing an authentication
+// via callback handler.
+type AuthFunc = func(*http.Request, *types.Message, types.HexBytes, string, int) AuthResponse
```

Comment:
use named parameters; what is the HexBytes? what is the string? what is the int?